ister_gui.py

```diff
@@ -2733,7 +2742,9 @@ def __init__(self, *args, **kwargs):
     def _init_actions(self):
         # auto install assumed
         keyboard_select = KeyboardSelection()
-        network_requirements = NetworkRequirements(1, 6)
+        network_requirements = NetworkRequirements(1, 6,
+                                                   self.args['contenturl'],
+                                                   self.args['versionurl'])
```

Comment:
Ah indeed, which reminds me since I have you here, what is your gui test method for kicking the tires outside of the unit tests?